projects/breach-notice-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/UpwAppointment.kt

```diff
@@ -15,12 +14,9 @@ class UpwAppointment(
     @Column(name = "upw_appointment_id")
     val id: Long,
 
+    @Column("contact_id")
     val contactId: Long,
```

Comment:
Think the JoinColumn should be here:
```suggestion
    @ManyToOne
    @JoinColumn("contact_id")
    val contact: Contact,
```

Reply:
done